app/apps/addresses/models.py

```diff
@@ -64,34 +69,40 @@ def __str__(self):
             )
         return self.bag_id
 
-    def get(bag_id):
+    def get_or_create(bag_id):
         return Address.objects.get_or_create(bag_id=bag_id)[0]
 
-    def save(self, *args, **kwargs):
+    def get_bag_address_data(self):
+        # When moving the import to the beginning of the file, a Django error follows:
```

Comment:
De naam is niet expliciet. GET suggereerst dat je de bag_address_data returnt.
Terwijl je in de functie potentieel zelfs een District aanmaakt.